test/test_chatcommands.py

```diff
@@ -278,16 +278,16 @@ def test_reject(monkeypatch):
 
     # Prevent from attempting to check privileges with Metasmoke
     monkeypatch.setattr(GlobalVars, "code_privileged_users", [])
-    assert chatcommands.reject('8888 "test"', original_msg=msg, alias_used="report").startswith("You need blacklist manager privileges")
+    assert chatcommands.reject('8888 "test"', original_msg=msg).startswith("You need blacklist manager privileges")
```

Comment:
You've removed `alias_used="report"`, instead of replacing it with `alias_used="reject"`. All of these should just replace "report" with "reject".

It looks like the `@command` decorator results in the command being actually called as a result of the `__call__` in chatcommunicate.py, which defaults `alias_used` to `None`. This appears to be a larger issue with how all of the commands are organized, which you should just work around here by providing `alias_used` in this line and below.

Reply:
And, I'm *way* behind.

Reply:
It seems you are reviewing an outdated version.